Tests/FullFunctionalTestPlan.xctestplan

```diff
@@ -98,6 +98,9 @@
         "PrivateBrowsingTest\/testiPadDirectAccessPrivateMode()",
         "PrivateBrowsingTest\/testiPadDirectAccessPrivateModeBrowserTab()",
         "PrivateBrowsingTestIpad\/testiPadDirectAccessPrivateMode()",
+        "ReaderModeTests\/testAddToReaderListOptions()",
+        "ReaderModeTests\/testAddToReadingList()",
+        "ReaderModeTests\/testLoadReaderContent()",
         "ReaderViewTest\/testAddToReaderListOptions()",
         "ReaderViewTest\/testAddToReadingList()",
```

Comment:
@dragosb01 should these old references be removed? just for clarity... (same for the other xctestpla files...

Reply:
@isabelrios updated